TLCGen/Views/Tabs/SpecialsTab/Tabs/PTPKoppelingenTabView.xaml

```diff
@@ -34,15 +38,85 @@
                             <DataGridTextColumn
                                 Visibility="{Binding Path=DataContext.ShowMultivalent,Source={x:Reference ProxyElement},Converter={StaticResource Bool2VisConv}}"
                                 Header="# Uit multiv." Binding="{Binding Path=AantalsignalenMultivalentUit,UpdateSourceTrigger=PropertyChanged}" />
-                            <DataGridTextColumn Header="Poort simulatie" Binding="{Binding Path=PortnummerSimuatieOmgeving,UpdateSourceTrigger=PropertyChanged}" />
+                            <DataGridTextColumn Header="Poort simulatie" Binding="{Binding Path=PortnummerSimuatieOmgeving,UpdateSourceTrigger=PropertyChanged}"/>
                             <DataGridTextColumn Header="Poort automaat" Binding="{Binding Path=PortnummerAutomaatOmgeving,UpdateSourceTrigger=PropertyChanged}" />
                             <DataGridTextColumn Header="Source" Binding="{Binding Path=NummerSource,UpdateSourceTrigger=PropertyChanged}" />
                             <DataGridTextColumn Header="Destination" Binding="{Binding Path=NummerDestination,UpdateSourceTrigger=PropertyChanged}" />
+                            <DataGridCheckBoxColumn Header="Dummy" Binding="{Binding Path=Dummy,UpdateSourceTrigger=PropertyChanged}" />
+                            <DataGridCheckBoxColumn Header="MaakIO" Binding="{Binding Path=MaakIO,UpdateSourceTrigger=PropertyChanged}" />
                         </DataGrid.Columns>
                     </DataGrid>
+                    <!--
```

Comment:
kan dit weg? of is het nog voor later gebruik? idem met code in commentaar hierboven